views/tutor-portal.handlebars

```diff
@@ -0,0 +1,104 @@
+
+<link rel="stylesheet" href="/css/tutorPortal.css" />
+
+
+<main class="container d-flex justify-content-center main-content">
+      <div
+        class="col teacup img-fluid mx-lg-auto xs-mx-auto"
+        style="width: 40rem"
+      >
+        <img src="./assets/tutorlist.png" class="" alt="tea cup" />
```

Comment:
You have to add and commit these image files for us to get them too

Reply:
i removed them on my mock up folder but not the real one so i will remove those